server/auth/sso/sso.go

```diff
@@ -114,7 +117,10 @@ func newSso(
 	if err != nil {
 		return nil, err
 	}
-
+	if c.Timeout.Duration <= 0 {
```

Comment:
Could we make the default 10h? Otherwise this will break for existing users?

Reply:
Did some changes for this.PTAL. Also E2E test started failing after this. Is is because of this change or something else?

Reply:
Done

Reply:
minor - just add `c.Timeout.GetDuration()`

Reply:
Not sure what you mean. Like this?

`expiry = expiry + c.Timeout.Duration`

Reply:
more like:

```golang
func (c Config) GetTimeout() time.duration {
  if c.Timeout > 0 {return c.Timeout}
  return 10 * time.Hour
}
```